howto/README.md

```diff
@@ -43,6 +43,10 @@ Here you'll find a list of How To guides that walk you through accomplishing spe
 * [Configure component secrets using Dapr secret stores](./setup-secret-store)
```

Comment:
Whilst you are updating this readme.md can you add the topic https://github.com/dapr/docs/tree/master/howto/configure-redis  which was moved by Aaron, but no link added in the readme.md. 
I suggest that you put a entry under **both** State Management heading and Pub/Sub heading called "Setup Redis as a state store" and  ""Setup Redis for pub/sub"

Reply:
It'll be a separate PR, we should really keep them separate.

Reply:
I'll do it right after this is merged.